src/services/sessionInvite.js

```diff
@@ -441,22 +442,28 @@ module.exports = class UserInviteHelper {
 					session.status = 'Invalid'
 					session.statusMessage = this.appendWithComma(session.statusMessage, ' Only 6 mentees are allowed')
```

Comment:
Why is this message hardcoded.
Isn't mentee count configurable ?

Reply:
Do i have to make it configurable @nevil-mathew ?

Reply:
Yes